installscripts/dockerfiles/jenkins-ce/Dockerfile

```diff
@@ -5,9 +5,13 @@ MAINTAINER JazzOSS Team
 # Switching to root to configure the image with system packages
 USER root
 RUN apt-get update && apt-get install -y vim curl sudo libtool autoconf make unzip rsync gcc autogen shtool pkg-config lsb-release python python-dev python-pip python-setuptools groff less && \
-curl https://bootstrap.pypa.io/get-pip.py | python && pip install --upgrade awscli && apt-get clean && pip install virtualenv && /usr/bin/easy_install virtualenv
```

Comment:
what was the issue here?

Reply:
Looks like changes related to python2 & python3 are combined into a single line (below).